File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/IncreaseStagedRolloutTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.GooglePlayPublisher;

public class IncreaseStagedRolloutTask extends BaseTask {
	
	public IncreaseStagedRolloutTask() {
		setDescription("Increase the fraction of users who should get the current staged rollout");
	}
	
	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.increaseStagedRollout(app);
	}
	
}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/IncreaseStagedRolloutTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

class IncreaseStagedRolloutTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().increaseStagedRollout(app)
    }

    init {
        description = "Increase the fraction of users who should get the current staged rollout"
    }
}
